cmd/tusd/cli/greeting.go

```diff
@@ -8,23 +8,26 @@ import (
 var greeting string
 
 func PrepareGreeting() {
+	// @todo: Introduce configurable metrics endpoint.
+	// See: https://github.com/tus/tusd/issues/52
```

Comment:
As it already acknowledges a configurable metrics endpoint via %s, rather than hardcoding in the text

Sent from mobile, pardon the brevity.

> On 14 jul. 2016, at 22:08, Marius <notifications@github.com> wrote:
> 
> In cmd/tusd/cli/greeting.go:
> 
> > @@ -8,23 +8,26 @@ import (
> >  var greeting string
> >  
> >  func PrepareGreeting() {
> > +	// @todo: Introduce configurable metrics endpoint.
> > +	// See: https://github.com/tus/tusd/issues/52
> Why did this end up here? :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>